cmd/server/config.go

```diff
@@ -0,0 +1,119 @@
+package main
+
+import (
+	"os"
+	"path/filepath"
+	"text/template"
+
+	"github.com/kubernetes-incubator/cri-o/server"
+	"github.com/opencontainers/runc/libcontainer/selinux"
+	"github.com/urfave/cli"
+)
+
+const (
+	ocidRoot   = "/var/lib/ocid"
+	conmonPath = "/usr/libexec/ocid/conmon"
+	pausePath  = "/usr/libexec/ocid/pause"
+)
+
+var commentedConfigTemplate = template.Must(template.New("config").Parse(`
+# Commented version of ocid configuration file. Generated automatically.
```

Comment:
Not sure we want this line.  If we use a this tool to auto generate /etc/ocid.conf, this line might make admins hesitant to modify the /etc/ocid.conf.  If we want to keep the line, it should be removed on install.